src/ert/_c_wrappers/enkf/config/enkf_config_node.py

```diff
@@ -396,36 +304,33 @@ def create_field(
             value_max = max_key
             truncation = truncation | EnkfTruncationType.TRUNCATE_MAX
 
-        config_node = cls._alloc_field_node(key, grid, forward_init)
-        if config_node is None:
-            raise ValueError(f"Failed to create FIELD node for:{key}")
+        config_node = cls._alloc_field_node(key, grid)
 
         cls.verifyValidFieldName(init_transform, "INIT_TRANSFORM")
         cls.verifyValidFieldName(input_transform, "INPUT_TRANSFORM")
         cls.verifyValidFieldName(output_transform, "OUTPUT_TRANSFORM")
 
-        if var_type_string == ConfigKeys.PARAMETER_KEY:
-            config_node._update_parameter_field(
-                ecl_file,
-                init_file_fmt,
-                truncation,
-                value_min,
-                value_max,
-                init_transform,
-                output_transform,
-            )
+        file_format_enum = (
+            FieldFileFormatType.ECL_KW_FILE_ALL_CELLS
+            if ecl_file
+            else FieldFileFormatType.FILE_FORMAT_NULL
+        )
 
-        elif var_type_string == ConfigKeys.GENERAL_KEY:
```

Comment:
What happened to the general var type? I guess we should leave some commit message in the history, and probably some text in PR description of why it is gone now.

Reply:
Was present in the commit message -- have added that info to the PR itself now. :+1: